### PR TITLE
Document the webhook delivery and retry logic

### PR DESCRIPTION
**Pull request by DXTimer**
This PR:

* Explain the delivery retry logic
* Explain the disable (suppression) logic

Sidebar Menu:
image

TC option added:
image

New documentation section:
image

### CODE REVIEW

File: content/v2/events.markdown [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+### Retry logic
+
+Across all environments, DNSimple attempts to deliver an event to your webhooks for up to 2 hours with exponential backoff. The delivery mechanism allows for up to 10 attempts within the allotted time.
+
+If your webhook endpoint has been disabled no further retry attempts are made for that webhook endpoint. However, if the endpoint is re-enabled before the next retry attempt DNSimple will continue to process the event delivery via the normal retry flow.
+
+### Suppression logic
+
+If your webhook endpoint has been misconfigured and has not responded with a 200 HTTP status code consistently for up to 15 attempts DNSimple will automatically suppress the webhook endpoint.
```
[the remark this answers is not in the document]
Comment (DXTimer) [reply content/v2/events.markdown]: Added in [1fbe04f](https://github.com/dnsimple/dnsimple-developer/pull/391/commits/1fbe04f2b8dc98c2f5cd524596d1c4a38997594e)

File: _redirects [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/v2/webhooks /v2/webhooks/webhooks-api
```
Comment (weppos) [inline _redirects:1]: Why this change? All the existing API endpoint documentations have the "API" suffix (which was missing here and I added), but none of the has the page end with -api.

See:
https://developer.dnsimple.com/v2/zones/records/
https://developer.dnsimple.com/v2/domains/

I don't see a reason to rename that page, it will become an exception.

**Review by weppos — CHANGES_REQUESTED** [review]
See my notes about the page rename.

File: _redirects (hunk 2 above)
Comment (DXTimer) [reply _redirects]: File name change reverted in [27346cb](https://github.com/dnsimple/dnsimple-developer/pull/391/commits/27346cba011641569440059f88acb8270d636d87)

**Review by weppos — APPROVED** [review]
